ref-tests/Cargo.toml

```diff
@@ -6,12 +6,13 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+candid = "0.6.7"
 delay = "0.3.0"
 ic-agent = { path = "../ic-agent" }
 ic-utils = { path = "../ic-utils", features = ["raw"] }
 ring = "0.16.11"
 tokio = "0.2.22"
+serde = { version = "1.0.101", features = ["derive"] }
```

Comment:
alphabetical order?

```suggestion
serde = { version = "1.0.101", features = ["derive"] }
tokio = "0.2.22"
```

Reply:
Done separately with the rest of the changes.